examples/posix_sockets/udp.c

```diff
@@ -33,6 +33,13 @@
 #include <sys/socket.h>
 #include <unistd.h>
 
+#ifdef SOCK_HAS_IPV6
+#include <net/ipv6/addr.h>     /* for interface parsing */
+#endif /* SOCK_HAS_IPV6 */
+#ifdef MODULE_GNRC_SOCK_UDP
+#include <net/gnrc/netif.h>    /* for finding default ipv6 interface */
```

Comment:
[Please use `"` for non-POSIX headers](https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md#includes).